apps/web/src/api/hooks/notification-templates/useCreateDigestDemoWorkflow.ts

```diff
@@ -26,7 +26,10 @@ export const useCreateDigestDemoWorkflow = () => {
       errorMessage('Failed to create Digest Workflow');
     },
   });
-  const { templates = [], loading: templatesLoading } = useTemplates();
+  const { templates = [], loading: templatesLoading } = useTemplates({
+    pageIndex: 0,
+    pageSize: 100,
+  });
```

Comment:
The templates are used here to check if the Digest Demo Workflow already exists or not. I changed this hook to reuse the same query cache as in the other places. See the one below.